src/cpp/types/topic/Topic.cpp

```diff
@@ -86,7 +95,7 @@ std::ostream& operator <<(
         std::ostream& os,
         const Topic& a)
 {
-    os << "Topic{" << a.topic_name() << ", " << a.topic_type() << "}";
+    os << "Topic{" << a.topic_name() << ";" << a.topic_type() << ";" << a.topic_with_key() << "}";
```

Comment:
NIT: Maybe print it as "keyed"|"no_key"??

Reply:
:+1: